pages/common/diffstat.md

```diff
@@ -0,0 +1,11 @@
+# diffstat
+
+> Make histogram of diff-output.
```

Comment:
Perhaps `Create a histogram from the output of the diff command` (with backticks around "diff") would read better?